peloton/update_project.py

```diff
@@ -928,10 +928,31 @@ def _run_sub_mutation(self, page_slice: slice) -> list:
         mutation = sgqlc.operation.Operation(github_schema_root.mutation_type)
         for ix in self._indexes[page_slice]:
             self._add_selector(mutation=mutation, index=ix)
-        result = run_operation(mutation)
-        # Fields are accessed from mutation results via their keys.
-        payloads = [result[a] for a in self._aliases[page_slice]]
-        return payloads
+
+        # Handle unpredictable GitHub timeouts.
+        attempts = 0
+        result = None
+        exception = None
+        while attempts < 5:
```

Comment:
Couldn't this be a for loop? Is there a reason to keep hold of the number of attempts (debugging maybe)? If so, I feel like you'd get a more accurate number by incrementing `attempts` before the try except block.